source/base/ByteBuffer.ooc

```diff
@@ -28,26 +28,29 @@ ByteBuffer: class {
 	size ::= this _size
 	_referenceCount: ReferenceCounter
 	referenceCount ::= this _referenceCount
-	init: func (=_pointer, =_size) {
+	_owner: Bool
```

Comment:
Really, `_pointer` is a bad name, too, so it should probably be `_buffer` and then you could call it `_freeBuffer`. On the other hand,

```ooc
if (this _ownsTheBuffer)
    // do stuff
```
reads a lot better than
```ooc
if (this _freeBuffer)
    // do stuff
```